eppo_client/bandit.py

```diff
@@ -104,101 +71,85 @@ def evaluate_bandit(
         flag_key: str,
         subject_key: str,
         subject_attributes: Attributes,
-        actions_with_contexts: List[ActionContext],
+        actions: ActionContexts,
```

Comment:
should we still keep this named`actions_with_contexts`?

Reply:
Thought about it -- given that we expect a dictionary now, I think it's more obvious that we need extra information to go with the key